tgg/misc_analyses/gnomad_small_variant_list_query.py

```diff
@@ -0,0 +1,2144 @@
+import argparse
```

Comment:
I think this script would benefit from a couple sentence overview of what it does and delivers, just so you dont have to read through the entire script. Thinking of whoever, if anyone, picks this up in the future.